test/fixtures/rules/std_rule_cases/L050.yml

```diff
@@ -0,0 +1,15 @@
+rule: L050
+
+test_pass_leading_whitespace_statement:
+  pass_str: "SELECT foo FROM bar\n"
+
+test_pass_leading_whitespace_comment:
+  pass_str: "/*I am a comment*/\nSELECT foo FROM bar\n"
+
```

Comment:
```suggestion
test_pass_leading_whitespace_comment:
  pass_str: "/*I am a comment*/\nSELECT foo FROM bar\n"

test_pass_leading_whitespace_inline_comment:
  pass_str: "--I am a comment\nSELECT foo FROM bar\n"

test_pass_leading_whitespace_inline_comment_hash:
  pass_str: "# I am a comment\nSELECT foo FROM bar\n"
  configs:
    core:
      dialect: bigquery

```

Reply:
All good additions, hadn't considered the bigquery hash :)